polaris/ocean/resolution.py

```diff
@@ -0,0 +1,22 @@
+def resolution_to_subdir(resolution):
```

Comment:
It seems this function isn't yet used by all tasks that could use it. Is that intentional?

Reply:
Yes.  This PR has all the framework (and ocean framework) changes but the changes to the test cases (anything that changes the worktree paths) is reserved for #117.  That way, we can do bit-for-bit testing with these changes.

This module could be moved to #117 if that seems better, since it's not yet used here.

Reply:
My assumption would be that you and @sbrus89 would review this and #117 together, and the separation was only to allow bit-for-bit testing of this branch along the way.

Reply:
I looked at https://github.com/E3SM-Project/polaris/pull/117 and still saw lines like this one that don't use `resolution_to_subdir`. I don't think it would change the worktree path, so it could be included in either PR. But if you prefer to leave this for later, that's also ok. I just wanted to make sure that was the intention. https://github.com/E3SM-Project/polaris/blob/923e3cd03872c4ab9a08aa80aacfa1462b778218/polaris/ocean/tasks/manufactured_solution/init.py#L40-L43

Reply:
> This module could be moved to https://github.com/E3SM-Project/polaris/pull/117 if that seems better, since it's not yet used here.

FYI, I did see that it was used here in #116:
https://github.com/E3SM-Project/polaris/blob/f3627c6881cc9d34f12077a2f0e1b55c1d29fac6/polaris/ocean/tasks/baroclinic_channel/__init__.py#L20

Reply:
Ah, you're right.  There are many opportunities to use this more widely.

Reply:
@cbegeman, this has now been addressed in #117: cosine bell (via `add_spherical_base_mesh_step()`), inertial gravity wave and manufactured solution all use `resolution_to_subdir()` to get their meshe names.

Reply:
Great! Thanks. Looks good by visual inspection.